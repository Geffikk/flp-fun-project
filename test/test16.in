S,X,Y
a,b
S
S->aXbX
S->abX
S->aXb
S->ab
X->aY
X->bY
X->a
X->b
Y->aY
Y->bY
