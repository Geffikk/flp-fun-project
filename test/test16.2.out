S,X,Y,a',<XbX>,<bX>,<Xb>,b'
a,b
S
S->a'<XbX>
a'->a
<XbX>->X<bX>
<bX>->b'X
b'->b
S->a'<bX>
S->a'<Xb>
<Xb>->Xb'
S->a'b'
X->a'Y
X->b'Y
X->a
X->b
Y->a'Y
Y->b'Y
